src/pages/plugins/index.md.ejs

```diff
@@ -2,17 +2,332 @@
 layout: content.html
 ---
 
+<%
+let cols = [
+{ label: 'col2'},
+{ label: 'col3'},
+{ label: 'col2&4'},
+{ label: 'crosscolumn'},
+{ label: 'copy voxgig/senecajs'},
+]
+%>
+
 # Plugins
-*Seneca* plugins enable you to build awesome microservices, incredibly fast.
+
+_Seneca_ plugins enable you to build awesome microservices, incredibly fast.
 Some plugins are maintained in-house, while others are provided by the community.
 
 On this page, we provide an `npm` link and total downloads per month for any plugins
 we find or publish ourselves, so you only need to look in one place.
 
+<%
+let plugins = {
+  plugingroup: [
+    {
+      name: 'plugin',
```

Comment:
use `name` as a string code value for internal references (eg # links)
wheres use `title` for the human visible text name of the group